dev/TabView/TabView.cpp

```diff
@@ -44,13 +54,48 @@ void TabView::OnApplyTemplate()
         }
         return scrollViewer;
     }());
+
+    m_addButton.set([this, controlProtected]() {
+        auto addButton = GetTemplateChildT<winrt::Button>(L"AddButton", controlProtected);
+        if (addButton)
+        {
+            // Do localization for the add button
+            auto addButtonName = ResourceAccessor::GetLocalizedStringResource(SR_TabViewAddButtonName);
+            winrt::AutomationProperties::SetName(addButton, addButtonName);
```

Comment:
>SetName [](start = 41, length = 7)

If someone overrides this in the template, it seems rude to stomp over it - I'd suggest only setting this if it's unset.

Reply:
OK, I'll check if it's empty first.